src/test/regress/custom_sql_schedule

```diff
@@ -3,3 +3,4 @@ test: prepared_statements_1 prepared_statements_2 prepared_statements_3
 test: prepared_statements_4
 test: ch_benchmarks_1 ch_benchmarks_2 ch_benchmarks_3
 test: ch_benchmarks_4 ch_benchmarks_5 ch_benchmarks_6
+test: intermediate_result_pruning_queries_1 intermediate_result_pruning_queries_2
```

Comment:
Minor: I skipped style related things as we will fix them in the main PR (such as no new line)